### PR TITLE
2 packages from andreypopp/ocamlformat-mlx at 0.26.2.0

### PR DESCRIPTION
**Pull request by andreypopp**
This pull-request concerns:
- `ocamlformat-mlx.0.26.2.0`: Auto-formatter for OCaml .mlx code
- `ocamlformat-mlx-lib.0.26.2.0`: OCaml .mlx Code Formatter

---
* Homepage: https://github.com/ocaml-mlx/ocamlformat-mlx
* Source repo: git+https://github.com/ocaml-mlx/ocamlformat-mlx.git
* Bug tracker: https://github.com/ocaml-mlx/ocamlformat-mlx/issues

---
:camel: Pull-request generated by opam-publish v2.4.0

### CODE REVIEW

File: packages/ocamlformat-mlx-lib/ocamlformat-mlx-lib.0.26.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml Code Formatter"
+description:
+  "OCamlFormat is a tool to automatically format OCaml code in a uniform style."
```
Comment (mseri) [inline packages/ocamlformat-mlx-lib/ocamlformat-mlx-lib.0.26.2.0/opam:5]: Are synopsis and description for this library correct?

File: packages/ocamlformat-mlx-lib/ocamlformat-mlx-lib.0.26.2.0/opam [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+]
+authors: [
+  "Andrey Popp <me@andreypopp.com>"
+  "Josh Berdine <jjb@fb.com>"
+  "Hugo Heuzard <hugo.heuzard@gmail.com>"
+  "Etienne Millon <etienne@tarides.com>"
+  "Guillaume Petiot <guillaume@tarides.com>"
+  "Jules Aguillon <jules@j3s.fr>"
+]
+homepage: "https://github.com/ocaml-mlx/ocamlformat"
```
Comment (mseri) [inline packages/ocamlformat-mlx-lib/ocamlformat-mlx-lib.0.26.2.0/opam:17]: The urls also seem incorrect: https://github.com/ocaml-mlx/ocamlformat-mlx

File: packages/ocamlformat-mlx/ocamlformat-mlx.0.26.2.0/opam [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+authors: [
+  "Andrey Popp <me@andreypopp.com>"
+  "Josh Berdine <jjb@fb.com>"
+  "Hugo Heuzard <hugo.heuzard@gmail.com>"
+  "Etienne Millon <etienne@tarides.com>"
+  "Guillaume Petiot <guillaume@tarides.com>"
+  "Jules Aguillon <jules@j3s.fr>"
+]
+homepage: "https://github.com/ocaml-mlx/ocamlformat"
+bug-reports: "https://github.com/ocaml-mlx/ocamlformat/issues"
```
Comment (mseri) [inline packages/ocamlformat-mlx/ocamlformat-mlx.0.26.2.0/opam:23]:
```suggestion
bug-reports: "https://github.com/ocaml-mlx/ocamlformat-mlx/issues"
```

File: packages/ocamlformat-mlx/ocamlformat-mlx.0.26.2.0/opam [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ocaml-mlx/ocamlformat.git"
```
Comment (mseri) [inline packages/ocamlformat-mlx/ocamlformat-mlx.0.26.2.0/opam:46]:
```suggestion
dev-repo: "git+https://github.com/ocaml-mlx/ocamlformat-mlx.git"
```

File: packages/ocamlformat-mlx/ocamlformat-mlx.0.26.2.0/opam [hunk 5]
```diff
@@ -0,0 +1,56 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Auto-formatter for mlx OCaml dialect code"
+description: """
+**ocamlformat-mlx** is a code formatter for mlx OCaml dialect. It comes with opinionated default settings but is also fully customizable to suit your coding style.
+
+- **Profiles:** ocamlformat offers profiles we predefined formatting configurations. Profiles include `default`, `ocamlformat`, `janestreet`.
```
[the remark this answers is not in the document]
Comment (andreypopp) [reply packages/ocamlformat-mlx/ocamlformat-mlx.0.26.2.0/opam:7]: Yes, it's a really small patch over ocamlformat, should be compatible.

**Comment by andreypopp** [comment]
Thanks for the @mseri, all should be fixed!

**Pull request merged**